.swiftpm/xcode/xcshareddata/xcschemes/KSCrash-Package.xcscheme

```diff
@@ -132,10 +132,10 @@
             </BuildableReference>
             <SkippedTests>
                <Test
-                  Identifier = "KSDebug_Tests">
```

Comment:
These tests were disabled for a different reason: test env on GH works differently somehow.